src/api-token-resolver.ts

```diff
@@ -4,12 +4,14 @@ export class ApiTokenResolver {
 
   public static getAccessToken(host: string, apiToken: string) {
   const base64ApiToken = new Buffer('apitoken:' + apiToken).toString('base64');
-  const options = {
+  let options = {
```

Comment:
Can stay `const`. Object assign will still work. const only means that the variable `options` cannot be reassigned, but the object behind it can still be mutated, as Object.assign does ;-)